core/matrix/ell_kernels.hpp

```diff
@@ -61,13 +62,43 @@ namespace kernels {
                           matrix::Dense<ValueType> *result,            \
                           const matrix::Ell<ValueType, IndexType> *source)
 
-#define GKO_DECLARE_ALL_AS_TEMPLATES                            \
-    template <typename ValueType, typename IndexType>           \
-    GKO_DECLARE_ELL_SPMV_KERNEL(ValueType, IndexType);          \
-    template <typename ValueType, typename IndexType>           \
-    GKO_DECLARE_ELL_ADVANCED_SPMV_KERNEL(ValueType, IndexType); \
-    template <typename ValueType, typename IndexType>           \
-    GKO_DECLARE_ELL_CONVERT_TO_DENSE_KERNEL(ValueType, IndexType)
+#define GKO_DECLARE_ELL_CONVERT_TO_CSR_KERNEL(ValueType, IndexType)  \
+    void convert_to_csr(std::shared_ptr<const DefaultExecutor> exec, \
+                        matrix::Csr<ValueType, IndexType> *result,   \
+                        const matrix::Ell<ValueType, IndexType> *source)
+
+#define GKO_DECLARE_ELL_MOVE_TO_CSR_KERNEL(ValueType, IndexType)  \
+    void move_to_csr(std::shared_ptr<const DefaultExecutor> exec, \
+                     matrix::Csr<ValueType, IndexType> *result,   \
+                     const matrix::Ell<ValueType, IndexType> *source)
```

Comment:
I don't think you have `move_to_csr` anymore, so you can delete it here as well.